code/style.css

```diff
@@ -1,58 +1,264 @@
+  font-size: 20px;
+  color: #000000;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.main-wrapper {
+  padding: 20px;
+  background-image: url("/code/assets/diamond-background.png");
+  background-size: 150px;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.header-container {
+  margin-top: 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #9dccc9;
+  padding: 20px;
+}
+
+.header-container h2 {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 100;
+  font-size: 28px;
+  padding-top: 10px;
+  margin: 0px;
+}
+
+.header-title {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-bottom: 1px solid #000000;
+  padding: 0px 30px;
+}
+
+.header-title h1 {
+  font-size: 50px;
+  margin-bottom: 10px;
+  margin-top: 5px;
+}
+
+.header-title img {
+  border-radius:45%;
+  padding: 5px;  
 }
 
-.cards {
-  margin: 50px auto;
+.cards {  
+  background: #cccccc;
+  margin: 30px auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  /* height: 900px; */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center; 
+}
+
+.card-container {
+  padding: 20px;
+  width: 100%;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  text-align: center;
+}
+
+.card-header-container {
+  padding: 10px;
+}
+
+.card-header-container h2 {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 100;
 }
 
 .killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+  width: 500px;
+  height: 550px;
   text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  margin-right: 30px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
+}
+
+.killer-deck p {
+  margin: 0px;
 }
 
 .weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
-  text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  width: 500px;
+  height: 550px;
+  margin-right: 30px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  background-repeat: repeat;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
 }
 
-.room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
+.weapon-deck p {
+  margin: 0px;
+}
 
+.room-deck {
+  width: 500px;
+  height: 550px;
   text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
+}
+
+.room-deck p {
+  margin: 0px;
+}
+
+.card-front {
+  display: block;
+  justify-content: center;
+  align-items: center;
+  margin-top: 30px;
+}
+
+.card-front img {
+  display: block;
+  margin-top: 50px;
+  margin-left: 165px;
```

Comment:
On my monitor (wide-screen), I noticed that the mag glass icon wasn't centered. If you replace the "margin-left: 165px" to
```
margin-left: auto; 
margin-right: auto;
```
everything looked a-ok.

Reply:
Amzing advice! Of course so simple. Will implement this!

Reply:
@EmOlsz I tried changing these margins to auto like Axel suggested, but it ended up shifting my magnify glass on each card out of the center where I had positioned them by using the specific pixels. Is there any way that you can use a standard,, like Axel suggested making them fix to the center of the card when being viewed on different screen sizes? Without having to adjust the margins for each screen size?